ext/TrixiConvexECOSExt.jl

```diff
@@ -67,6 +67,48 @@ function stability_polynomials!(pnoms, consistency_order, num_stage_evals,
     end
 end
 
+# Specialized form of the stability polynomials for fourth-order PERK schemes.
+function stability_polynomials_PERK4!(pnoms, num_stage_evals,
+                                      normalized_powered_eigvals,
+                                      gamma, dt, cS3)
+    num_eig_vals = length(pnoms)
```

Comment:
This comment also applies to the original `stability_polynomials` function, but shouldn't we pass `num_eig_vals `as the function's argument so we don't have to write this line of code? As I take a closer look, I now realize that we always have this value and writing just looks a bit inconsistent. If we did not want to pass it and preferred to recalculate it whenever needed, we should not make `num_eig_vals` an argument in `bisect_stability_polynomials`.